modules/placement-driver/README.md

```diff
@@ -0,0 +1,13 @@
+# Placement driver
+
+The module responses for location of leaseholder node in the replication group. The placement driver decides a node which can hold a lease 
+and approves with the replication group.
+
+## Leaseholder management
+Placement driver manager depends on Metastorage and a logical topology. The manager listens change of group members (due to a particular 
```

Comment:
Let's also rephrase this, placement driver logically depends on the assignments and logical topology, there's no big deal that assignments are stored inside ms.